src/seo/getAlternateLanguageLinks.ts

```diff
@@ -0,0 +1,78 @@
+import type { NewsroomLanguageSettings } from '@prezly/sdk';
+
+import type { AlternateLanguageLink } from '../components-nextjs/PageSeo';
+import { getFallbackLocale, LocaleObject } from '../intl';
+
+import { createAlternateLanguageLink } from './utils/createAlternateLanguageLink';
+import { getDefaultHrefLang } from './utils/getDefaultHrefLang';
+import { mapLanguagesToLocales } from './utils/mapLanguagesToLocales';
+
+export function getAlternateLanguageLinks(
+    languages: NewsroomLanguageSettings[],
+    getTranslationUrl: (locale: LocaleObject) => string | undefined,
+    createHref: (locale: LocaleObject, translationUrl: string) => string,
```

Comment:
These two params look too similar. Can we not combine then into one function `generateUrl(contentLocale, hrefLan) => string | undefined` ?